proposals/lip-0053.md

```diff
@@ -88,7 +87,7 @@ Setup to gather the required mainchain information:
 Create a cross-chain update transaction for a given height `h1`:
 
 * Find a [signed certificate][lip-0061] in the mainchain block headers for a height (say `h2`) higher or equal to `h1`. This will be used as the `certificate` property of the transaction.
```

Comment:
It would be nice to link this section here: https://github.com/LiskHQ/lips/blob/main/proposals/lip-0061.md#selecting-optimal-certificates-for-cross-chain-certification

Reply:
added